rest-api-spec/src/main/resources/rest-api-spec/test/cluster.health/10_basic.yml

```diff
@@ -132,4 +132,150 @@
   - is_true: indices
   - is_true: indices.test_index.shards
 
+---
+"cluster health with closed index (pre 8.0)":
+  - skip:
+      version: "8.0.0 - "
+      reason:  "closed indices are replicated starting version 8.0"
+
+  - do:
+      indices.create:
+        index: index-1
+        body:
+          settings:
+            index:
+              number_of_replicas: 0
+
+  - do:
+      cluster.health:
+        wait_for_status: green
+  - match:     { status:     green }
+
+  - do:
+      indices.create:
+        index: index-2
+        body:
+          settings:
+            index:
+              number_of_replicas: 50
+
+  - do:
+      cluster.health:
+        wait_for_status: yellow
+        wait_for_no_relocating_shards: true
```

Comment:
Also wait_for_no_initializing-shards? Seems we do so in ClusterHealthIT, so thought this should be the same?